main.py

```diff
@@ -49,17 +55,28 @@ def validate_id(cls, values):
 products = []
 
 
-# эдитпоин для добавления нового продукта
-@app.post('/products')
-async def add_product(product: Product):
-    try:
-        products.append(product.dict())
-        return {'status': 'success', 'data': product.dict()}, 201
-    except ValidationError as e:
-        return {'status': 'error', 'message': e.errors()}, 400
-
-
-# эдитпоинт для получения списка всех продуктов
 @app.get('/products/all')
 async def get_all_products():
     return {'status': 'success', 'data': products}, 200
+
+
+# Define the endpoint to add a new Product to MongoDB
+@app.post("/products/add")
```

Comment:
Remember what we discussed about REST. You don't need to have any verbs in URL. You already have them as HTTP methods. So, what is better:

POST /products

or

POST /products/add

?